.circleci/config.yml

```diff
@@ -65,7 +65,7 @@ jobs:
       - run:
           name: "Run WebdriverIO V6"
           command: |
-            cd webdriverio
+            cd webdriverio/web/framework
```

Comment:
@nikolay-advolodkin 

I know I wasn't in Florida, but I don't like this structure of AND web AND framework. This will be a lot of overhead and a lot of double work. Web could also still be emu sim or real device. 

I don't mind having a structure like this:

```
--frameworkA
   |__selenium
   |__appium
--frameworkB
   |__selenium
   |__appium
--featureA
   |__selenium
   |__appium
--featureB
   |__selenium
   |__appium
```

where frameworkA/B is for example Protractor/Nightwatch/WebdriverIO which hold the best practices on:
- how to run locally
- how to run on Sauce VM-EMUSIM-RDC
- uses best practices

FeatureA/B is something where we use one of the best frameworks to explain features, in JS case it will be WebdriverIO

Reply:
@wswebcreation this is the right way to organize unfortunately. It's not as much work as you think because not all code exists everywhere all the time. However, when someone wants wdio v5 with jasmine versus wdio v6 with cucumber, we need to know exactly where to put the code. The recommended layout of our repositories does that. It's less of a question of is it maintainable, we as a team already determined the answer is yes. My question to you is does it make sense for JS?